README.md

```diff
@@ -8,23 +8,53 @@ please reach out to us in [gitter](https://gitter.im/vinyldns/Lobby).
 To run, `pip install vinyldns-python` and then:
 
 ```python
->>> from vinyldns_python import *
->>> local_client = client.VinylDNSClient("ApiEndpoint", "UserAccessKey", "UserSecretKey")
+>>> from vinyldns.client import VinylDNSClient
+>>> local_client = VinylDNSClient("ApiEndpoint", "UserAccessKey", "UserSecretKey")
 >>> local_client.list_zones()
 >>>
 >>> # If all of the following environments are set
 >>> # - VINYLDNS_API_URL
 >>> # - VINYLDNS_ACCESS_KEY_ID
 >>> # - VINYLDNS_SECRET_ACCESS_KEY
->>> from vinyldns_python import *
->>> local_client = client.VinylDNSClient.from_env()
+>>> from vinyldns.client import VinylDNSClient
+>>> local_client = VinylDNSClient.from_env()
 >>> local_client.list_zones()
 ```
 
+## Contributing
+
+**Requirements**
+
+* `python3`
```

Comment:
Are we "strongly encouraging" python3 but still plan to maintain compatibility with python2?